### ISSUE TITLE
Can we use rounded corners?

### ISSUE THREAD

**Issue by Snugug** [IBM-Watson/design-library#129]
I've been thinking about this a lot recently, and I've got a general question I'm going to throw out there.

I understand that in the Design Language, rounded corners are forbidden in order to make our designs seem more machine-like (this can probably be extended to animations too, but let's stick with rounded corners). But we want Watson to feel [humane](https://github.com/IBM-Watson/design-library/wiki/Intro-to-WDL), not just mechanical. Take your laptops; the only hard corners in the whole thing are for the screen; everything else has a rounded corner. Keys, trackpad, shell, even power cube and Mag Power plug, all rounded corners. Remotes, designed for direct human interaction, all have rounded corners. Hell, even giant industrial blow molders have rounded corners

![Blow Molding Machine](http://image.made-in-china.com/2f0j00fMHakpVKHsqL/5-Gallon-Semi-Automatic-Stretch-Blow-Molding-Machine-CE-L-BS511-.jpg)

So basically, my question is, should we allow rounded corners in our designs? I'm not suggesting like 50% border radius on everything, but just enough to take the edge off of a hard corner much like what GitHub does here. In the name of being a little softer, a little more humane than just a machine.

**Comment by joshkimmell** [comment]
I'd like to consider it, but now isn't the time. If you want to have that discussion with Adam Cutler, we can have it later in the week. IBM Design is building a brand that eventually all IBM will fall under and they're pretty adamant about the sharp corners.

**Comment by bobbinrobyn** [comment]
Short answer: no, not right now. I think we can close this issue.

**Comment by joshkimmell** [comment]
The answer is no. Let's see what the products do without and we can iterate later.

**Comment by joshkimmell** [comment]
yes to close... no to rounded corners.
